code/modules/projectiles/projectile/beams.dm

```diff
@@ -27,17 +27,29 @@
 	wound_bonus = -30
 	bare_wound_bonus = 40
 
-//overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
-/obj/projectile/beam/laser/hellfire
+
+/obj/projectile/beam/laser/hellfire //overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
 	name = "hellfire laser"
-	wound_bonus = 0
-	damage = 25
+	wound_bonus = -5
+	damage = 20
 	speed = 0.6 // higher power = faster, that's how light works right
 
 /obj/projectile/beam/laser/hellfire/Initialize(mapload)
 	. = ..()
 	transform *= 2
 
+/obj/projectile/beam/laser/hellfire/ignite //Hellfire laser but ignites, but does less wounding/damage
+	name = "burn laser"
+	wound_bonus = -30 //You can ignite
+	damage = 10
```

Comment:
```suggestion
	damage = 20
```

Reply:
What bug is this fixing? You have been asked specifically to not use code review to make balance suggestions